### PR TITLE
Add a method to TestContext to check if is IPv6

### PR DESCRIPTION
**Pull request by aojea**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug

/kind cleanup

> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

This is a followup based on the following conversation

https://github.com/kubernetes/kubernetes/pull/81750#discussion_r318052060

to move the e2e ClusterIsIPv6() function to a method of TestContext

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

This has to be merged after #82014

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by neolit123** [comment]
@aojea 
PTAL at the `pull-kubernetes-bazel-test` failures.

having this as a member function as @pohly recommended seems better to me.
(but whether TestContext being a public variable is sane, is another topic... :))
/approve

/assign @oomichi 
for /test approval.

thanks.

**Review by oomichi — COMMENTED** [review]
This seems to make the framework clean for me.

/lgtm
/approve
/retest

**Review by oomichi — COMMENTED** [review]
/lgtm cancel

File: test/e2e/framework/util.go [hunk 1]
```diff
@@ -430,11 +430,6 @@ func getDefaultClusterIPFamily(c clientset.Interface) string {
 	return "ipv4"
 }
 
-// ClusterIsIPv6 returns true if the cluster is IPv6
-func ClusterIsIPv6() bool {
-	return TestContext.IPFamily == "ipv6"
-}
-
```
[the remark this answers is not in the document]
Comment (aojea) [reply test/e2e/framework/util.go]: > This has to be merged after #82014

:)

**Comment by aojea** [comment]
/retest
/hold cancel

The PR that fixed the unit test failures was merged

**Pull request merged**